feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -285,11 +284,31 @@ func verifyReadReceiveMatch(expected_update []*p4_v1.Update, received_entry *p4_
 
 	matches := 0
 	for _, table := range received_entry.Entities {
-		if diff := cmp.Diff(expected_update[0].Entity.Entity, table.Entity, protocmp.Transform()); diff != "" {
-			glog.Errorf("Table entry diff (-want +got): \n%s", diff)
-			continue
+		//add a switch case for the captured table entry
+		//Switch case to remove the counter_data and meter_config from table entry returned by p4rt server
+		var tableMap map[string]interface{}
```

Comment:
"any" is now preferred over interface{}

Reply:
Hi Dennis, 

I will make the switch in the next commit. Thanks.